docs/dev/naming_conventions.rst

```diff
@@ -82,32 +87,23 @@ Core layer
     * ``core_ferc714__hourly_demand_pa``,
     * ``core_ferc1__yearly_plant_in_service``.
 
-Output layer
-^^^^^^^^^^^^
-* Assets in this layer use the well modeled tables from the Core layer to construct
-  wide and complete tables suitable for users to perform analysis on. This layer
-  contains intermediate tables that bridge the core and user-facing tables.
-* Naming convention: ``out_{source}__{asset_type}_{asset_name}``
-* ``source`` is optional in this layer because there can be assets that join data from
-  multiple sources.
-* ``asset_type`` is also optional. It will likely describe the frequency at which
-  the data is reported (annual/monthly/hourly).
 
-Intermediate Assets
-^^^^^^^^^^^^^^^^^^^
-* Intermediate assets are logical steps towards a final well-modeled core or
-  user-facing output asset. These assets are not intended to be persisted in the
-  database or accessible to the user. These assets are denoted by a preceding
-  underscore, like a private python method. For example, the intermediate asset
-  ``_core_eia860__plants`` is a logical step towards the
-  ``core_eia860__entity_plants`` and ``core_eia860__scd_plants`` assets.
-  ``_core_eia860__plants`` does some basic cleaning of the ``raw_eia860__plant``
-  asset but still contains duplicate plant entities. The computation intensive
-  harvesting process deduplicates ``_core_eia860__plants`` and outputs the
-  ``core_eia860__entity_plants`` and ``core_eia860__scd_plants`` assets which
-  follow Tiny Data standards.
-* Limit the number of intermediate assets to avoid an extremely
-  cluttered DAG. It is appropriate to create an intermediate asset when:
+Core Layer (Intermediate Assets)
```

Comment:
I think the title should stay Intermediate Assets because they are a type of asset that can exist in the Core and Output layer.

Reply:
Ok that's fine with me!